functions/src/index.ts

```diff
@@ -107,3 +114,13 @@ async function handleUpdateDocument(
     logs.error(e as Error)
   }
 }
+
+/**
+ * addSearchableFields
+ */
+export async function addSearchableFields(): Promise<void> {
+  if (config.searchableFields?.length != 0) {
+    const index = await client.getOrCreateIndex(config.meilisearchIndex)
+    await index.updateSearchableAttributes(getSearchableFields())
```

Comment:
```suggestion
    const searchableFields = getSearchableFields()
    await index.updateSearchableAttributes(searchableFields)
```

Just for  readability, but if you prefer like you did initially no problem

Reply:
No it better this way